client/components/LicenseSelect/LicenseSelect.tsx

```diff
@@ -33,10 +33,7 @@ const LicenseSelect = (props: Props) => {
 	const { communityData } = usePageContext();
 	const currentLicense = getLicenseBySlug(pubData.licenseSlug)!;
 	const pubCopyrightYear = getPubCopyrightYear(pubData);
-	let pubPublisher = communityData.title;
-	if (communityData.id === '78810858-8c4a-4435-a669-6bb176b61d40') {
-		pubPublisher = 'Massachusetts Institute of Technology';
-	}
+	const pubPublisher = communityData.publishAs || communityData.title;
```

Comment:
Since this is being rewritten in two places, and seems important to name, I'd suggest a utility function in `utils/license.js` or maybe a `utils/community.ts` with the following signature:

```ts
getPublisherString(community: Pick<Community, 'publishAs' | 'title'>): string
```